config/environments/production.rb

```diff
@@ -1,6 +1,21 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  host = '<your heroku app>.herokuapp.com'
```

Comment:
using environment variable to save host value

Reply:
sorry i do not understand, what should i change?

Reply:
if you deploy your app using heroku, you must create heroku app then paste your link heroku'app here. ex: host = 'example.herokuapp.com'
https://devcenter.heroku.com/articles/creating-apps#creating-a-named-app
But in this chapter, you do not need to deploy, so ignore this comment